src/index.js

```diff
@@ -1 +1,56 @@
 import './style.css';
+
+// POPULATE LIST
+
+import populateList from './module/showList.js';
+
+async function populate() {
+  const requestURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/IV1251GkSsG9dGnKrPGk/scores/';
+  const request = new Request(requestURL);
+
+  const response = await fetch(request);
+  const allPlayers = await response.json();
+
+  populateList(allPlayers);
+}
```

Comment:
- [ ] Could you please update your function to use ES6 arrow functions in line with [the project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md)? 😊

   ![image](https://github.com/eamador1/leaderboard/assets/107952724/af44041f-9545-4a73-a165-21758813846f)